src/Plugin_Command.php

```diff
@@ -735,10 +741,21 @@ protected function get_item_list() {
 				'file'               => $file,
 				'auto_update'        => in_array( $file, $auto_updates, true ),
 				'author'             => $details['Author'],
+				'tested_up_to'       => '',
 				'wporg_status'       => $wporg_info['status'],
 				'wporg_last_updated' => $wporg_info['last_updated'],
 			];
 
+			if ( $this->check_headers['tested_up_to'] ) {
+				// Include information from the plugin readme.txt headers.
+				$plugin_readme = WP_PLUGIN_DIR . '/' . $name . '/readme.txt';
```

Comment:
Should we normalize file path here? `Utils\normalize_path()`? What do you think?

Reply:
May be we should handle `readme.md` also as it is supported in the directory.

Reply:
@ernilambar thanks for the feedback, and pointing out that function. Yes, I think we need to normalize the path. I will make the necessary update. 

We could probably add support for the `readme.md`. But which file takes precedence assuming there are two readme files within the plugin? Is it the `readme.md` or `readme.txt`?

Reply:
`readme.txt` takes precedence